src/App.tsx

```diff
@@ -1063,7 +1153,23 @@ function App(props: RouteComponentProps) {
                                     Genome View
                                 </button>
                                 <button
-                                    className="navigation-button navigation-button-variant"
+                                    className="navigation-button split-right"
+                                    tabIndex={1}
+                                    data-bs-toggle="modal"
+                                    data-bs-target="#genome-view-modal"
+                                >
+                                    <svg className="button question-mark" viewBox={ICONS.QUESTION_CIRCLE_FILL.viewBox}>
+                                        <title>Question Mark</title>
+                                        {ICONS.QUESTION_CIRCLE_FILL.path.map(p => (
+                                            <path fill="currentColor" key={p} d={p} />
+                                        ))}
+                                    </svg>
+                                </button>
+                            </div>
+                            <div className="navigation-button-container split navigation-button-variant">
+                                <button
+                                    className="navigation-button split-left"
+                                    tabIndex={1}
```

Comment:
Should the modal view be shown when a user clicks on the question mark of either genome view or variant view in the normal mode? It seems like the modal view is only shown in the minimal mode, and clicking on the question marks does not do anything on the normal mode.